Azure-Samples/modern-data-warehouse-dataops
# feat: fixing main deployment issues

# Type of PR

- Documentation changes
- Code changes

## Purpose
It fixes the following issues:
- Failure due to change in behavior of workspace startup, instead there is a pause in the script to authenticate to the databricks workspace
- Dedicated Synapse pool deployment failure
- Upgrade Databricks CLI as the Databricks configuration script was failing with the previous CLI

## Does this introduce a breaking change? If yes, details on what can break
It might introduce a breaking change as the Databricks CLI is being upgraded. 
The deployment needs to be tested end-to-end.

## Author pre-publish checklist

## Validation steps
- Clone the repository
- Launch the devcontainer
- Verify within the devcontainer that the databricks CLI version is Databricks CLI v0.232.1 or above by running databricks --version
- Follow the pre-requirements to deploy the sample
- Run the ./deploy.sh script
- Verify that the deployment is successful in the 3 environments
- Within the databricks workspaces, verify that the cluster was created successfully (including the ddo library), and that the job executes successfully - all the tables are created under the Hive Metastore

## Issues Closed or Referenced

- Closes #630 
- Closes #778 
- Closes #763
